run-checks

```diff
@@ -292,14 +293,20 @@ if [ "$UNIT" = 1 ]; then
     command -v go
     go version
 
+    tags=()
```

Comment:
This is a bit hard to read but it seems like it's just the way it is with shell :/ Slightly sad that there seems to be no better way. Thanks for finding a solution for this!